api-flask/app/main.py

```diff
@@ -44,15 +44,19 @@ def _calculate_stats(zones, geotiff, stats, prefix, group_by, geojson_out):
 @timed
 def stats():
     """Return zonal statistics."""
-    geotiff_url = request.form.get('geotiff_url', None)
-    zones_url = request.form.get('zones_url', None)
+    # Accept data as json or form.
+    data = request.get_json() or request.form
+    geotiff_url = data.get('geotiff_url', None)
+    zones_url = data.get('zones_url', None)
     if not (geotiff_url and zones_url):
-        raise Exception('geotiff_url and zones_url are both required.')
-
-    geojson_out = request.form.get('geojson_out', 'False')
-    geojson_out = strtobool(geojson_out)
+        logger.error('Received {}'.format(data))
+        return Response(
+            response='501: geotiff_url and zones_url are both required.',
+            status=501
```

Comment:
This isn't a 5xx response (server issue), it's a 4xx (client issue). I'd probably either use 400 (general malformed client request) or 422 (unprocessable entity)